src/components/theme-switcher/theme-switcher.module.css

```
@@ -0,0 +1,37 @@
+.root {
+	position: relative;
+	color: var(--token-color-foreground-primary);
+	width: 100%;
+}
+.select {
```

🎨 Previous comment like this hasn't been resolved yet but it's not showing in the diff anymore, so just making it again for posterity's sake!

```suggestion
}

.select {
```